sm_project/cl_app/views.py

```diff
@@ -6,13 +6,21 @@
 # Renders a view of all the checklists that the user has.
 def checklist_index(request):
     if request.user.is_authenticated:
+        if request.user.has_temp_checklist():
+            request.user.get_temp_checklist().delete()
         return render(request, 'cl_app/checklist_index.html', {'user_checklists': request.user})
     else:
         return redirect('user_app:login')
     
 # Renders a view of the add checklist screen and allows the user to create a new checklist.
 def add_checklist(request):
     if request.user.is_authenticated:
+        item_form = ChecklistItemForm()
+        if request.user.has_temp_checklist():
+            temp_checklist = request.user.get_temp_checklist()
+        else:
+            temp_checklist = Checklist(checklist_title = ("Temp" + str(request.user.id)), creator = request.user)
```

Comment:
```suggestion
            temp_checklist = Checklist(checklist_title = (f"Temp{request.user.id}"), creator = request.user)
```

Using f-strings over the + operator is the new suggested way to concatenate strings. Easier to read and doesn't require the cast to str()...